IPtProxy.go/controller.go

```diff
@@ -339,8 +339,7 @@ func (c *Controller) Start(methodName string, proxy string) {
 	if proxy != "" {
 		proxyURL, err = url.Parse(proxy)
 		if err != nil {
-			ptlog.Errorf("Failed to parse proxy address: %s", err.Error())
-			return
+			ptlog.Warnf("Failed to parse proxy address: %s", err.Error())
```

Comment:
Are you sure you want to do this? It will result in the transport being run without a proxy.

It also made me wonder if we should be returning errors from `Start`. Do you think that would be easier for the callers of the library to handle? Otherwise it's hard to tell whether or not the transport successfully started.

Reply:
🤔 returning errors. Definitely something to think about.
Well, error codes, at least, I'd say, as error handling in Objective-C, Swift and the JVM are *very* different than in Go.